src/classes/issues-processor.ts

```diff
@@ -74,7 +75,7 @@ export class IssuesProcessor {
 
   constructor(options: IIssuesProcessorOptions) {
     this.options = options;
-    this.client = getOctokit(this.options.repoToken);
+    this.client = getOctokit(this.options.repoToken, {}, retry);
```

Comment:
```suggestion
    this.client = getOctokit(this.options.repoToken, undefined, retry);
```
As the second parameter of the function getOctokit() is optional, it means that it has union type: `OctokitOptions | undefined`. Would it be more plain to set it to undefined instead of empty object {}, If we don't specify any Octokit options ?

Reply:
It is a matter of taste. I prefer to clearly state "we do not pass any options" instead of passing something vague and ambiguous. `undefined` is returning value of some expression, issuing it seems to be weird for me.

Reply:
For now with undefined and empty it should be the [same](https://github.com/actions/toolkit/blob/d2b7d85e7cb61467395604aae1b84439c320cb84/packages/github/src/utils.ts#L35). But I would prefer to change it to `undefined` because if default option will be changed it won't get affect for us because an empty object is a truthy value.